wagtail/wagtailcore/blocks/field_block.py

```diff
@@ -262,7 +262,9 @@ def get_searchable_content(self, value):
 
 class RichTextBlock(FieldBlock):
 
-    def __init__(self, required=True, help_text=None, **kwargs):
+    def __init__(self, required=True, help_text=None, hallo_config=None, **kwargs):
+
+        self.hallo_config = hallo_config
```

Comment:
Hello, @HitDaCa! Thanks for contributing to Wagtail!

I think will be better to rename `hallo_config` to `hallo_plugins` for [consistency with Hallo](https://github.com/bergie/hallo/tree/1.1.1#plugins).

What do you think about it?